host/src/types/routes.ts

```diff
@@ -6,4 +6,14 @@ export const AuthRoutes = {
   resetPassword: "/reset-password",
 } as const;
 
+export const DashboardRoutes = {
+  home: "/dashboard/home",
+  listShortLet: "/dashboard/list-shortlet",
```

Comment:
We will be using one route for shorlets - adding a new one and showing available shorlets

```suggestion

```